src/sentry/dynamic_sampling/rules/helpers/utils.py

```diff
@@ -0,0 +1,10 @@
+from typing import Any
+
+from django.conf import settings
+
+from sentry.utils import redis
+
+
+def get_redis_client_for_ds() -> Any:
+    cluster_key = getattr(settings, "SENTRY_DYNAMIC_SAMPLING_RULES_REDIS_CLUSTER", "default")
+    return redis.redis_clusters.get(cluster_key)
```

Comment:
Can we put this into the higher-level `utils.py` file? If we keep creating utils files scoped to the smallest scope possible, we might incur into many files that can be confusing. What do you think?

Reply:
yeah it's a good idea.

Reply:
oh you know what, do you mean to move to:
https://github.com/getsentry/sentry/blob/24cfb08af67aed8e99b7052c85ef63443829c16a/src/sentry/dynamic_sampling/rules/utils.py#L166-L169

Reply:
Yep, exactly

Reply:
done.